writer/proto/resource.go

```diff
@@ -115,16 +137,17 @@ func GeneratedResourceMessage(r *parser.ParsedResource) (*builder.MessageBuilder
 			},
 		))
 	}
-	mb.SetOptions(
-		&descriptorpb.MessageOptions{},
+	//mb.SetOptions(
```

Comment:
nit: delete all this commented code? I'm not sure how useful it is.

Reply:
Sounds good! Didn't quite know what this was.